apiary.apib

```diff
@@ -1557,6 +1561,14 @@ Retrieves all the different Clearings card types for the current client.
         Filter by date range.
     + shop_id (optional, string)
         Filter by shop id
+    + stablishment_id (optional, string)
+        Filter by stablishment id
+    + priority (optional, integer)
+        Filter by priority
+    + created_at_from (optional, date)
```

Comment:
There's the standard ISO 8601 format used in most APIs and JSON do you think it is still not clear? Now that I think about it we are not using the standard in other parameters of the API, we should